packages/near-swap-provider/lib/NearSwapProvider.ts

```diff
@@ -13,7 +13,7 @@ import {
 
 import Bytecode from './bytecode'
 
-const CONTRACT_CODE = 'wnGY4a+YYkfbkBqYqyhdQjjtHRl9Auyh7yxCGtXL8JI='
+const CONTRACT_CODE = 'jrBWhtpuyGJ44vtP+Ib+I32tuUUtfKQBuBdQ8y3M6Ro='
```

Comment:
@kraikov how can one audit/review this contract code?

Reply:
The bytecode is taken from the near contract - In order to validate the change, you can clone the repo, install the dependencies in both the root and contract forlders and run npm build.
This will generate a bytecode.json file that includes the bytecode generated after compiling the contract.

Not sure how this is generated. Probably it's baseXX of the hash of the bytecode. However, once you verify the bytecode is good, you can create swap transaction, read the metadata of the transaction from Near and there you'll find transaction.code